### PR TITLE
Feat: added navigation, grid comp. and links

### PR DESCRIPTION
**Pull request by paarthbajaj**
screencapture-127-0-0-1-5500-navigation-html-2022-03-29-22_55_15
screencapture-127-0-0-1-5500-grid-html-2022-03-29-22_54_41

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Good strcuture
Good color combination
Good naming

**Review by Raghav888 — APPROVED** [review]

File: navigation.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            overflow: hidden;
+          "
+          sandbox="allow-scripts allow-same-origin"
+        >
+        </iframe>
+
+        <h4 class="heading-3">Usage</h4>
+        <ul>
+          <li>
+            Semantically your navbar should be under header tag. Nag tag tells
```
Comment (harshitadaswani) [inline navigation.html]: typo: `nav`

**Review by harshitadaswani — APPROVED** [review]
The code looks structured and the UI also looks appealing. However, there is a typo that you can correct and then go ahead with the merge.

File: navigation.html (hunk 1 above)
Comment (paarthbajaj) [reply navigation.html]: Thank you for pointing it out. Correcting it.

**Pull request merged**